src/man/sssd.conf.5.xml

```diff
@@ -359,19 +359,19 @@
                     <varlistentry>
                         <term>try_inotify (boolean)</term>
                         <listitem>
-                            <para>
+                            <para condition="have_inotify">
```

Comment:
Wouldn't it be better to condition entire `varlistentry` (if possible)?

Reply:
Please notice the last paragraph is not optional. It is always displayed and it states:
> Note: this option will have no effect on platforms where inotify is unavailable. On these platforms, polling will always be used.

The idea is to tell the users that don't have inotify why they can't use this option. This text was present before my modifications.